src/utils/helper.ts

```diff
@@ -11,27 +11,22 @@ export const scrollToTop = (scrollContentId: string) =>
 
 export const orderCardIfNeeded = (cards: TSortableItems[], userCardConfig: string[] | undefined) =>
   userCardConfig
-    ? cards.sort((a, b) => {
-        return userCardConfig.indexOf(a.id) > userCardConfig.indexOf(b.id) ? 1 : -1;
-      })
+    ? cards
+        .slice()
+        .sort((a, b) => (userCardConfig.indexOf(a.id) > userCardConfig.indexOf(b.id) ? 1 : -1))
     : cards;
 
 export const getOrderFromAntdValue = (order: string): SortDirection =>
   order === 'ascend' ? SortDirection.Asc : SortDirection.Desc;
 
-export const formatQuerySortList = (sorter: SorterResult<any> | SorterResult<any>[]) => {
-  const sorters = (isArray(sorter) ? sorter : [sorter]).filter(
-    (sorter) => !!sorter.column || !!sorter.order,
-  );
+export const convertToArray = <T>(value: T | T[]) => (isArray(value) ? value : [value]);
```

Comment:
Personally, I think `[l].flat()` is more idiomatic. Besides, I don't see the point of making a function out of it.

Reply:
Removed the convertToArray function and used ```[l].flat()``` instead